src/Riok.Mapperly/AnalyzerReleases.Shipped.md

```diff
@@ -106,3 +106,11 @@ RMG044  | Mapper   | Warning  | An ignored enum member can not be found on the s
 RMG045  | Mapper   | Warning  | An ignored enum member can not be found on the target enum
 RMG046  | Mapper   | Error    | The used C# language version is not supported by Mapperly, Mapperly requires at least C# 9.0
 RMG047  | Mapper   | Error    | Cannot map to member path due to modifying a temporary value, see CS1612
+
+## Release 3.1
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+RMG048  | Mapper   | Error    | Cannot use multiple attributes in the same assembly
```

Comment:
IMO `multiple attributes` can be misleading as only multiple attributes of the type `DefaultMapperAttribute` are not supported.

Reply:
Is this diagnostic even requried as AllowMultiple is false on the attribute, isn't there already a diagnostic by the csharp compiler if it is applied multiple times?

Reply:
Changed the text of the diagnostic. It is required, since no diagnostic is created from the csharp compiler.